src/app/components/AutoComplete.svelte

```diff
@@ -24,13 +36,18 @@
   // Index of focus element
   let index = -1;
 
+  onMount(async () => {
+    results = await worker.getDataFromStorage();
+  });
+
   // Input to search
-  function onChange(event: Event) {
+  async function onChange(event: Event) {
```

Comment:
lol why does this keep getting `async`?

```suggestion
 function onChange(event: Event) {
```